application/src/test/java/org/opentripplanner/graph_builder/module/OsmBoardingLocationsModuleTest.java

```diff
@@ -54,7 +54,7 @@ static Stream<Arguments> herrenbergTestCases() {
           .map(VertexLabel::osm)
           .collect(Collectors.toSet())
       ),
-      Arguments.of(true, Set.of(VertexLabel.osm(3223067049L), VertexLabel.osm(768590748)))
```

Comment:
I believe this is intended, but can you explain what is going on?

Reply:
One of the optimizations was adding a threshold for convexity. 3 points along a straight line were randomly interpreted to define a convex or concave angle, because of inaccuracy of computations.  Only true concave points matter in visibility computations. Herrenberg test platform lost one unnecessary fake visibility point which is good.